cmd/vcsim/main.go

```diff
@@ -35,12 +37,15 @@ func main() {
 	flag.IntVar(&model.Pool, "pool", model.Pool, "Number of resource pools per compute resource")
 
 	isESX := flag.Bool("esx", false, "Simulate standalone ESX")
+	isTLS := flag.Bool("tls", false, "Enable TLS")
```

Comment:
Yeah we're using too many flags packages stdlib, go-flags, urfave/cli and govmomi has its own.  The only user facing one in VIC is urfave/cli (via vic-machine).  We should discuss the topic outside of this PR.